packages/blast-tests/e2e-tests/run-single-test.sh

```diff
@@ -17,38 +17,30 @@ fi
 
 if [[ $1 == 'node-start-status.test.sh' || $1 == 'node-start-custom-accounts.test.sh' ]]; then
     if [[ `docker ps` =~ $CONTAINER_NAME ]]; then
-        echo 'Node is started. Your node will be stopped and started again after the test is executed...'
-        $compose down &> /dev/null && sleep 5
-        node_stopped=true
+        echo "We are detecting a running node! In order to run the integration tests we need a fresh instance of Blast Node. Do you want us to restart it?"
```

Comment:
I am not sure whether I like the message style using "we". I would go with something like `A running node is detected. The end-to-end tests require a fresh instance of a Blast node. Do you want to restart it?`